include/tvm/relax/expr.h

```diff
@@ -985,6 +985,11 @@ constexpr const char* kComposite = "Composite";
 constexpr const char* kPartitionedFromPattern = "PartitionedFromPattern";
 /*! \brief The required workspace for an external function. */
 constexpr const char* kWorkspaceSize = "WorkspaceSize";
+/*! \brief Indicate whether the function is pure (has no visible side effects for any input). */
+constexpr const char* kIsPure = "IsPure";
+/*! \brief Indicate whether the function should be considered pure even if it contains
+ * an impure call. */
+constexpr const char* kForcePure = "ForcePure";
```

Comment:
One minor nit: we now hav two kinds of patterns for attribute name

- stl_style: `global_symbol`, `tir.noalias`
- CamelStyle: as in this case.

Would be great for us to consolidate a bit to a single style. Some of the CameStyle were inherited from relay. To be consistent with some of the more frequently used annotation (like global symbol)

Reply:
Okay, I'll rename it. I'm not a fan of CamelCase in general ;)

Reply:
I've eliminated the need to use these attributes, so this will no longer be an issue.